### ISSUE TITLE
Neat Stuff

### ISSUE THREAD

**Issue by cwoll87** [facundolinlaud/long-ago#2]
Hey i just wanted to let you know, that i really like what you did here, even if its old :). :+1
Maybe its time to migrate to a higher libgdx Version an start using kotlin.

**Comment by facundolinlaud** [comment]
hey thanks, it means a lot!

the project is using libgdx 1.9.8 which I think is the latest stable version available. in regards to kotlin, that is a nice advice, it would surely speed up the development process a lot! java gets too stiff sometimes...

**Comment by cwoll87** [comment]
You are welcome mate!
Guess i got something confused, obviously 1.9.8 is very up to date.
I tried to update all deps on your Project and use gradle 5.1.1 Kotlin and new Plugins,
but the GTW Plugin is to outdated to make it run.

**Comment by facundolinlaud** [comment]
yeah, I remember trying to get gwt to work without any luck. I specifically had trouble with the Jackson library not being compatible with gwt, I guess it was because Jackson uses reflection. There are some alternatives but I got bored after a time and sticked to the desktop wrapper. However, I'm planning on adding the libgdx android wrapper. I don't think it will take that much refactoring to get it to work properly
